docs/support_playbook.md

```diff
@@ -332,17 +332,17 @@ If the course doesn't exist in the previous cycle we'll need them to confirm the
 If the course details have changed from one cycle to another, provider users should contact support to request the changes. To confirm a deferral through the console:
 
 ```ruby
-zendesk_url = ZENDESK_URL
+zendesk_url = 'https://becomingateacher.zendesk.com/agent/tickets/529071'
 support_user = SupportUser.find_by(email_address: YOUR_SUPPORT_EMAIL)
 
-application_choice_id = APPLICATION_CHOICE_ID
-provider_code = PROVIDER_CODE
-new_course_code = NEW_COURSE_CODE
-new_course_recruitment_cycle_year = NEW_COURSE_RECRUITMENT_CYCLE_YEAR
-new_site_code = NEW_SITE_CODE
-new_study_mode = NEW_STUDY_MODE # part_time or full_time
-
+application_choice_id = 740269
 application_choice = ApplicationChoice.find(application_choice_id)
+provider_code = "N91"
+new_course_code = "W9X1"
+new_course_recruitment_cycle_year = application_choice.current_recruitment_cycle_year
```

Comment:
I'm not sure this is always correct -- if they are deferring, is it the _next_ cycle year relative to the original choice?  Honestly don't know for sure.